Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecorator.java

```diff
@@ -91,7 +91,7 @@ private String getKeepAliveProperty() {
         try {
             return getPropertyAccessor().getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.KEEP_ALIVE_PROP);
         } catch (PropertyAccessException ex) {
-            LOG.warn("Unable to access property: {}", NhincConstants.KEEP_ALIVE_PROP, ex.getLocalizedMessage());
+            LOG.warn("Unable to access property: {}", NhincConstants.KEEP_ALIVE_PROP, ex);
```

Comment:
try to use LOG.warn("Unable to access property: {} {}", NhincConstants.KEEP_ALIVE_PROP, ex.getLocalizedMessage(),ex);